CHANGELOG.md

```diff
@@ -207,6 +207,22 @@ If there are links back to radios or checkboxes components in your error summary
 
 [Pull request #1426: Make radios and checkboxes components easier to link to from error summary](https://github.com/alphagov/govuk-frontend/pull/1426)
 
+### Update the markup for tabs
+
+You do not need to do anything if you're not using using Nunjucks macros.
+
+If you are not using the Nunjucks macros, remove the `govuk-tabs__tab--selected` from the first tab's link, and add the new class `govuk-tabs__list-item--selected` to its parent list item instead:
```

Comment:
Just a style tweak. I've assumed this should say if you *are* using Nunjucks macro, because line 212 already says 'if you are not'.

```suggestion
If you're using Nunjucks macros, remove the `govuk-tabs__tab--selected` class from the first tab's link, then add the `govuk-tabs__list-item--selected` class to the link's parent list item.

For example:
```

Reply:
D'oh! It's entirely the other way around, good spot.

Reply:
Line 212 also says 'using using'!